lab_2_keywords_cooccurrence/main.py

```diff
@@ -123,8 +225,34 @@ def extract_candidate_keyword_phrases_with_adjoining(candidate_keyword_phrases:
 
     In case of corrupt input arguments, None is returned
     """
-    pass
-
+    if not (candidate_keyword_phrases and phrases):
+        return None
+    #тут будет проверка на тип
+    possible_pairs = {}
+    for i in range(len(candidate_keyword_phrases) - 1):
+        pair = candidate_keyword_phrases[i], candidate_keyword_phrases[i + 1]
+        if pair not in possible_pairs:
+            possible_pairs[pair] = 0
+        else:
+            possible_pairs[pair] += 1
+    appropriate_pairs = []
+    for pair in possible_pairs:
+        x = []
+        if possible_pairs[pair] > 1:
+            for i in pair:
+                i = ' '.join(i)
+                x.append(i)
+            if x:
```

Comment:
can you clarify how this condition can be unsatisfied?

Reply:
now i understand that this condition is redundant. During working on this function i was confused because of its complextity  and had to add this condition. I fixed it and deleted this